### ISSUE TITLE
Option to vertically scroll instead of zoom on timeline

### ISSUE THREAD

**Issue by jrpomeroy** [almende/vis#2341]
For my timeline use case being able to use the mouse wheel to scroll vertically is much more important than zooming.  However, it seems even if I disable zooming the mouse wheel still doesn't work for scrolling vertically.  In that case the mouse wheel just does nothing.  Ideally setting zoomKey would enable zoom while holding the key down but allow vertical scrolling when the key is not held down.  Am I missing any way to make this work?

**Comment by yotamberk** [comment]
Try verticalScroll:true and zoomKey:'ctrlKey'

**Comment by jrpomeroy** [comment]
Sorry, I should have mentioned I already tried playing with `verticalScroll: true`.  All it seems to do is add a visible scrollbar to the left of the groups, but does not make the mouse wheel scroll the timeline vertically when the mouse is over the content.  The timeline does scroll vertically when the mouse is over the groups, just not the content.  But then it works that way without verticalScroll set to true, so in that respect I'm confused what use verticalScroll provides other than making the scrollbar always visible.

**Comment by yotamberk** [comment]
Well, if you set a maxHeight, verticalScroll and zoomKey, it is quite useful... 
If that's not what youre looking for, please supply a jsbin and lets okay on an exampless until we get what you are looking for.

**Comment by jrpomeroy** [comment]
Basically what I was hoping for is a way to make the timeline simply let the mouse wheel events propagate up to the browser so that it could handle those events as it normally would.  I was able to make this work by removing the 'mousewheel' event listener after creating my timeline.  It's not ideal since this will not allow zooming at all, but it's probably good enough for my case.
```
this.timeline = new vis.Timeline(...);
this.timeline.range.body.emitter.off('mousewheel');
```

**Comment by yotamberk** [comment]
I'm really not sure what you mean. The behavior you're describing works for me. What browser are you using? Can you supply a jsbin to show your example and we can try and see if this is a bug or simply a miss understanding of the options?

**Comment by halli-pronestor** [comment]
Same problem here i think. If you take this example http://visjs.org/examples/timeline/other/groupsPerformance.html and add zoomKey: 'ctrlKey' to options. Then make the browser window small enough so it only shows Trucks 1 and 2. Then the page does not scroll when mouse is over timeline (vis-panel vis-center). The page only scrolls when mouse is over group names.

**Comment by mojoaxel** [comment]
There is also a [question on stackoverflow](http://stackoverflow.com/questions/40142243/timeline-vis-js-scroll-up-and-scroll-down-to-move-view-from-left-to-right).

**Comment by halli-pronestor** [comment]
I was using 4.15 where page scrolling worked, but didnt work after upgrading to 4.17.

**Comment by mojoaxel** [comment]
@halli-pronestor That worked before? What was the control-key? Maybe it's a bug?

**Comment by halli-pronestor** [comment]
@mojoaxel By control-key you mean zoomKey:'ctrlKey' ? My problem i described above, with 4.17 it does not scroll when mouse is over the vis-center, but works if i swap it out with 4.16.1.

**Comment by yotamberk** [comment]
I'm on it. PR will be ready by the weekend

**Comment by halli-pronestor** [comment]
Little more info from me. It is of course mouse wheel scroll. I upgraded from 4.15 to 4.17 to get a fix on a similar problem where finger scroll did not work over the vis-center on devices.

**Comment by yotamberk** [comment]
Fixed with #2301

**Comment by zhu-judy** [comment]
> Fixed with #2301

the bug is in 4.21.0again
